doc/sphinx/source/references.bib

```diff
@@ -190,3 +189,15 @@ @phdthesis{whitingStabilizedFEM1999
   langid = {english},
   school = {Rennselear Polytechnic Institute},
 }
+
+@article{shurSTG2014,
+  title = {Synthetic {{Turbulence Generators}} for {{RANS-LES Interfaces}} in {{Zonal Simulations}} of {{Aerodynamic}} and {{Aeroacoustic Problems}}},
+  author = {Shur, Michael L. and Spalart, Philippe R. and Strelets, Michael K. and Travin, Andrey K.},
+  year = {2014},
+  journal = {Flow, Turbulence and Combustion},
+  volume = {93},
+  number = {1},
+  pages = {63--92},
+  doi = {10/f58pzd},
```

Comment:
I don't understand how doi.org/10/f58pzd resolves, but the publisher's site says
```suggestion
  doi = {10.1007/s10494-014-9534-8},
```

Reply:
It's a [short DOI](https://shortdoi.org/), which is an official way of getting shorter DOI links. Nicer for actual paper publications (less noise in citation, easier to re-type if copy/paste isn't an option), but I think either works perfectly fine here.